linux_os/guide/system/software/gnome/gnome_login_screen/gnome_gdm_disable_unattended_automatic_login/rule.yml

```
@@ -21,13 +21,15 @@ rationale: |-
 severity: high
 
 identifiers:
+    cce@sle12: CCE-83245-1
     cce@sle15: CCE-85723-5
 
 references:
     disa: CCI-000366
     nist: CM-6(b),CM-6.1(iv)
     srg: SRG-OS-000480-GPOS-00229
     stigid@sle15: SLES-15-040430
+    stigid@sle12: SLES-12-010380
```

```suggestion
    stigid@sle12: SLES-12-010380
    stigid@sle15: SLES-15-040430
```

we now have a gating test that detects if references are sorted and in this case sle12 needs to come first.

For future reference, you can run these two commands from the project's directory to sort references automatically:

`python utils/rule_dir_json.py`
`python utils/fix_rules.py sort_subkeys -y`